source/_integrations/ukraine_alarm.markdown

```diff
@@ -0,0 +1,30 @@
+---
+title: Ukraine Alarm
+description: Instructions on how to integrate Ukraine Alarm within Home Assistant.
+ha_category:
+  - Other
+ha_release: 2022.6
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@PaulAnnekov'
+ha_domain: ukraine_alarm
+ha_platforms:
+  - binary_sensor
+ha_integration_type: integration
+---
+
+The Ukraine Alarm integration uses the [Ukraine Alarm](https://www.ukrainealarm.com/) web service to offer air-raid siren notifications. The integration will create 4 binary sensors for your selected region in Ukraine:
+
+- Air
+- Artillery
+- Urban Fights
+- Unknown
```

Comment:
Is it explained on the official site what the unknown alarm represents?

Reply:
No. I will ask support. I think it means "something is flying but we don't know what".